src/replicator.rs

```diff
@@ -104,8 +102,7 @@ impl Replicator {
             cluster_info_w.set_leader(leader_info.id);
         }
 
-        let (entry_window_sender, entry_window_receiver) = channel();
-        let store_ledger_stage = StoreLedgerStage::new(entry_window_receiver, ledger_path);
+        let (entry_window_sender, _entry_window_receiver) = channel();
```

Comment:
What's going on here? Looks like either a memory leak or an unused sender masquerading a used one.

Reply:
Yea, I will remove this.